IdentityCore/src/validation/MSIDAADAuthority.m

```diff
@@ -64,7 +64,7 @@ - (nullable instancetype)initWithURL:(nonnull NSURL *)url
     self = [self initWithURL:url context:context error:error];
     if (self)
     {
-        if (rawTenant && self.tenant.type != MSIDAADTenantTypeIdentifier)
+        if (rawTenant)
```

Comment:
ahhh, here's where this code got removed. I was wondering before how it worked :)